index.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+    font-family: 'League Spartan';
+    src: url('fonts/subset-LeagueSpartan-Bold.eot');
+    src: url('fonts/subset-LeagueSpartan-Bold.eot?#iefix') format('embedded-opentype'),
+        url('fonts/subset-LeagueSpartan-Bold.woff') format('woff'),
+        url('fonts/subset-LeagueSpartan-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+/* Header styling */
+.wrapper {
+    width: 1630px;
+    margin: 12px auto;
```

Comment:
Set the 12px margin on the header instead, because it will cause you troubles afterwards (I'll explain why)